### ISSUE TITLE
Reference section: "back to text" button next to the citation when a reference is clicked

### ISSUE THREAD

**Issue by mattiker** [openlibhums/janeway#3242]
**Is your feature request related to a problem? Please describe.**
References don't have a jump mark back to the text if you click on the href, notes do have a circumflex for that.

**Describe the solution you'd like**
Inject a "back to text" button next to the citation when it is clicked, using some javascript.

**Describe alternatives you've considered**
Readers should be able to jump back to the last position of the page after hitting a reference-href.

**Comment by StephDriver** [comment]
branch has been merged into master
